quartz/cli/handlers.js

```diff
@@ -2,6 +2,7 @@ import { promises } from "fs"
 import path from "path"
 import esbuild from "esbuild"
 import chalk from "chalk"
+import replace from "replace-in-file"
```

Comment:
Hmm can we just manage this through a toml or yaml file instead of doing this, similar to `lazy-lock.json` or any type of lock.

also remove deps 😄

Reply:
Could you elaborate? I'm open to other suggestions but wasn't getting much in the discord

Reply:
similar to how `lazy.nvim` did this, or package.json, since we already have a "plugin spec", we can just lock all of the dependences with github URL and revision in a lock file, so no need to do any codegen in ts.

we can just manage and modify the lockfile.

Reply:
In L584, thats runtime TS codegen if I understand this correctly?

Reply:
Not exactly. It's just taking what the developer told me to use from the plugin spec file (now a json, but will be reworked on your suggestion) and adding it. If I was creating `default_config_ts` from the content of the plug-in's ts file itself, that would be codegen. L565

Reply:
got it. Lemme take a look at what this replace function does.

Reply:
Just replaces everything matched by the regex in the supplied files with the passed-in string. I could *technically* do this without dependencies and just nodejs file syntax, but it only runs on a CLI command, so I wasn't too concerned about server load.